packages/lib-classifier/src/store/SubjectViewerStore.js

```diff
@@ -63,12 +69,21 @@ const SubjectViewer = types
         self.fullscreen = false
       },
 
-      onSubjectReady () {
+      onSubjectReady (event) {
+        const { target = {} } = event || {}
+        const {
+          clientHeight = 0,
+          clientWidth = 0,
+          naturalHeight = 0,
+          naturalWidth = 0
+        } = target
+        self.dimensions.push({ clientHeight, clientWidth, naturalHeight, naturalWidth })
         self.ready = true
```

Comment:
Maybe use `self.ready = self.dimensions.length === subject.locations.length` here to account for subjects with multiple linked files.

Reply:
Yes, we'll want to do something like that when multi-image subject support is added. Could you note that in an issue? We have a milestone for implementing the other subject viewers that it could be linked to.

Reply:
Sure. There's also audio projects where each subject has two locations: one audio, one image; and video projects like Chimp & See where the first location is a video and the rest of the locations array are images.

HTML5 media don't fire a load event so we'll have to come back to this anyway.